Doc/library/ast.rst

```diff
@@ -359,14 +340,11 @@ Expressions
 
    .. doctest::
 
-        >>> print(ast.dump(ast.parse('-a'), indent=4))
-        Module(
-            body=[
-                Expr(
-                    value=UnaryOp(
-                        op=USub(),
-                        operand=Name(id='a', ctx=Load())))],
-            type_ignores=[])
+        >>> print(ast.dump(ast.parse('-a', mode='eval'), indent=4))
```

Comment:
**This** example should use the "exec" mode.

Reply:
Could you elaborate? The difference between the two trees does not add anything that stands out to me:

```
>>> print(ast.dump(ast.parse('-a', mode='eval'), indent=4))
Expression(
    body=UnaryOp(
        op=USub(),
        operand=Name(id='a', ctx=Load())))
>>> print(ast.dump(ast.parse('-a', mode='exec'), indent=4))
Module(
    body=[
        Expr(
            value=UnaryOp(
                op=USub(),
                operand=Name(id='a', ctx=Load())))],
    type_ignores=[])

```

Reply:
It is an example for the `Expr` node.

Reply:
:man_facepalming: Whops! I need more coffee.